### PR TITLE
[docs] Add organization token

### PR DESCRIPTION
**Pull request by EdvinasDaugirdas**
Add documentation for organization token.

### CODE REVIEW

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 1]
```diff
@@ -3,23 +3,43 @@ title: 'Token'
...
-> <a href="#authenticating-with-api-token">See below</a> for more information.
+> For Single Sign-On (SSO) users, `k6 login cloud` requires a k6 Cloud account with an email and a password. If you don't want to create a password (using [Forgot Password](https://app.k6.io/account/forgot)), you can authenticate via API token. See [Authenticating with API token](#authenticating-with-api-token) for more information.
 
 > #### Docker Users
 >
-> If you're running k6 in a Docker container you'll need to make sure that the k6 config file where the k6 Cloud API authentication information is stored to is persisted via a Docker volume to the host machine, using the `-c/--config PATH/TO/CONFIG_FILE` CLI flag, e.g. `docker run --rm -i -v /path/on-host:/path/in-container/ grafana/k6 login cloud -c /path/in-container/config.json`.
+> If you're running k6 in a Docker container you'll need to make sure that the k6 config file where the k6 Cloud API authentication information is stored to is persisted via a Docker volume to the host machine, using the `-c/--config PATH/TO/CONFIG_FILE` CLI flag. Example:
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:18]:
```suggestion
> If you're running k6 in a Docker container, make sure that the k6 config file where the k6 Cloud API authentication information is stored is persisted via a Docker volume to the host machine, using the `-c/--config PATH/TO/CONFIG_FILE` CLI flag. Example:
```

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 2]
```diff
@@ -3,23 +3,43 @@ title: 'Token'
...
+```bash
+docker run --rm -i -v /path/on-host:/path/in-container/ grafana/k6 login cloud -c /path/in-container/config.json
+```
 
 > #### Integrating with CI
 >
 > If you are integrating k6 into your CI pipeline, we recommend using one of the token methods to authenticate and not exposing your username/password within your CI configuration files or as variables.
 
+## Account API token
+Account level API token provides API access to your account with k6. To generate the token select **API token** menu item under **Account settings**, where you'll be able to copy, see and regenerate the token.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:28]:
```suggestion
An account-level API token provides API access to your account with k6. To generate the token, go to **Account settings** and select **API token**.
From there you can copy, see, and regenerate the token.
```

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 3]
```diff
@@ -3,23 +3,43 @@ title: 'Token'
...
+
+![account token view](./images/04-Token/account-api-token-view.png)
+
+## Organization API token
+Organization level API token provides API access to your organization with k6. To generate a token select **API token** menu item under **Organization settings**, where you'll be able to create, see and regenerate the tokens.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:33]:
```suggestion
An organization-level API token provides API access to your organization with k6. To generate a token, go to **Organization settings** and select **API token**.
From there you can create, see, and regenerate the tokens.
```

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 4]
```diff
@@ -3,23 +3,43 @@ title: 'Token'
...
+Maximum amount of tokens an organization can create is 5.
+
+![organization token view](./images/04-Token/organization-api-token-view.png)
+
+> #### Access to organization settings
+>
+> Only [organization admins](/cloud/project-and-team-management/members/#admin) can access organization settings view.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:40]:
```suggestion
> Only [organization admins](/cloud/project-and-team-management/members/#admin) can access organization settings.
```

**Review by MattDodsonEnglish — COMMENTED** [review]
Made some suggested language changes

**Comment by MattDodsonEnglish** [comment]
Now that I've read the whole document again, I see that what you wrote sounds reasonable. Maybe as I know the k6 Cloud and its docs better, I'll have a different idea of how to organize this information by different use cases. But I don't want to let "perfect be the enemy of good."

> Was thinking about adding a link to `API Token as an environment variable` section in `Organization API token`.

👍

**Comment by ana-yankova** [comment]
@MattDodsonEnglish I added more information about the org token. Could you please have another look?

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 5]
```diff
@@ -3,23 +3,46 @@ title: 'Token'
...
 >
-> If you're running k6 in a Docker container you'll need to make sure that the k6 config file where the k6 Cloud API authentication information is stored to is persisted via a Docker volume to the host machine, using the `-c/--config PATH/TO/CONFIG_FILE` CLI flag, e.g. `docker run --rm -i -v /path/on-host:/path/in-container/ grafana/k6 login cloud -c /path/in-container/config.json`.
+> If you're running k6 in a Docker container, make sure that the k6 config file where the k6 Cloud API authentication information is stored is persisted via a Docker volume to the host machine, using the `-c/--config PATH/TO/CONFIG_FILE` CLI flag. Example:
+```bash
+docker run --rm -i -v /path/on-host:/path/in-container/ grafana/k6 login cloud -c /path/in-container/config.json
+```
 
 > #### Integrating with CI
 >
 > If you are integrating k6 into your CI pipeline, we recommend using one of the token methods to authenticate and not exposing your username/password within your CI configuration files or as variables.
```
Comment (EdvinasDaugirdas) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:25]: @ana-yankova do you think it makes sense to add a link here to `Organization API token` section here for visibility purposes?

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 6]
```diff
@@ -3,22 +3,45 @@ title: 'Token'
 excerpt: 'How to authenticate with k6 Cloud token'
 ---
 
-You'll need to authenticate to use k6 Cloud, whether it's for streaming results or running tests in the cloud. Your Auth Token enables the interaction with k6 Cloud using the k6 CLI or through the REST API. [Get your Auth Token](https://app.k6.io/account/api-token).
+You'll need to authenticate to use k6 Cloud, whether it's for streaming results or running tests in the cloud. Your API token enables the interaction with k6 Cloud using the k6 CLI or through the REST API.
 
-Below are some examples on how to utilize the token to authenticate.
+In k6 Cloud there are two ways how you can generate an API Token, via account or organization settings. These tokens differ in the kind of access they provide - [account level API token](#account-api-token) will grant access through your account with k6, while [organization level API token](#organization-api-token) will grant organization-wide access for actions not tied to a user.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:8]:
```suggestion
You can generate in API token for k6 Cloud in two ways, each with different levels of access:
- [Account-level API tokens](#account-api-token) grant access through your account with k6
- [Organization-level API tokens](#organization-api-token) grant organization-wide access for actions not tied to a user.    
```

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 7]
```diff
@@ -3,22 +3,45 @@ title: 'Token'
 excerpt: 'How to authenticate with k6 Cloud token'
 ---
 
-You'll need to authenticate to use k6 Cloud, whether it's for streaming results or running tests in the cloud. Your Auth Token enables the interaction with k6 Cloud using the k6 CLI or through the REST API. [Get your Auth Token](https://app.k6.io/account/api-token).
+You'll need to authenticate to use k6 Cloud, whether it's for streaming results or running tests in the cloud. Your API token enables the interaction with k6 Cloud using the k6 CLI or through the REST API.
 
-Below are some examples on how to utilize the token to authenticate.
+In k6 Cloud there are two ways how you can generate an API Token, via account or organization settings. These tokens differ in the kind of access they provide - [account level API token](#account-api-token) will grant access through your account with k6, while [organization level API token](#organization-api-token) will grant organization-wide access for actions not tied to a user.    
+
+Below you'll find instructions on how to generate API tokens and examples on how to utilize them.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:10]:
```suggestion
To generate API tokens, you can follow these steps or adapt these examples.
```

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 8]
```diff
@@ -3,22 +3,45 @@ title: 'Token'
...
+> If you are integrating k6 into your CI pipeline, we recommend using an [organization level API token](#organization-api-token) to authenticate and not exposing your username/password within your CI configuration files or as variables. Alternatively you can use an [account level API token](#account-api-token) with a user created specially for the purpose of running CI tests.
+
+## Account API token
+An account-level API token provides API access to your account with k6. To generate the token, go to **Account settings** and select **API token**.
+From there you can copy, see, and regenerate the token.
+
+![account token view](./images/04-Token/account-api-token-view.png)
+
+## Organization API token
+An organization API token provides organization-wide API access with k6. It is intended for actions that are not tied to a specific user like running tests from a CI platform. To generate a token, go to **Organization settings** and select **API token**.
```
[the remark this answers is not in the document]
Comment (MattDodsonEnglish) [reply src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:34]: Made a slight guess in this edit. Please check before committing.

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 9]
```diff
@@ -3,22 +3,45 @@ title: 'Token'
...
+From there you can create, see, and regenerate the tokens. Maximum amount of tokens an organization can create by default is 5.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:35]:
```suggestion
From there you can create, see, and regenerate the tokens.
The maximum amount of tokens that an organization can create is 5 by default.
```

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md [hunk 10]
```diff
@@ -3,22 +3,45 @@ title: 'Token'
...
+
+Note that when using an organization API token you must always specify the project the test runs will be created under. Learn how to set the `Project ID` [here](https://k6.io/docs/cloud/creating-and-running-a-test/cloud-tests-from-the-cli/#running-tests-under-a-different-project-than-your-default-one).
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:37]:
```suggestion
Note that when using an organization API token, you must specify the project where the test runs will be created.
Read about [how to set the `Project ID`](https://k6.io/docs/cloud/creating-and-running-a-test/cloud-tests-from-the-cli/#running-tests-under-a-different-project-than-your-default-one). 
```

**Review by MattDodsonEnglish — COMMENTED** [review]
I made some minor edits to make the language more precise, but I think it looks more accurate and complete now. :+1:

File: src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md (hunk 6 above)
Comment (ana-yankova) [reply src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:9]: > You can generate in API token for k6 Cloud in two ways, each with different levels of access:

I might be wrong as a non-native speaker, but a `level of access` sounds to me like with one token you get more permissions than with the other which is not necessarily true. It's just that there are different usages for the two tokens and the authentication happens on a different level, either through a user where the user's permissions are checked or globally for the organisation (than an admin must have created and provided the token).

Comment (MattDodsonEnglish) [reply src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md:9]: You're completely right! I changed it again using the terms you just used:

> You can generate two types of API tokens for k6 Cloud, depending on how you want to authenticate:

**Comment by MattDodsonEnglish** [comment]
@ana-yankova  Looks good to me. Can we merge?

**Pull request merged**